src/gui/uithemecommon.h

```diff
@@ -82,6 +82,8 @@ inline QHash<QString, UIThemeColor> defaultUIThemeColors()
         {u"TransferList.MissingFiles"_s, {Color::Primer::Light::dangerFg, Color::Primer::Dark::dangerFg}},
         {u"TransferList.Error"_s, {Color::Primer::Light::dangerFg, Color::Primer::Dark::dangerFg}},
 
+        {u"ProgressBar.Chunk"_s, {{}, {}}},
```

Comment:
I wonder if it should be called just "ProgressBar". IMO, it is unlikely other parts of progress bar should be customizable.